docs/User_Guide_BPMN.md

```diff
@@ -1,4 +1,4 @@
-# ![camunda logo](http://camunda.github.io/camunda-bpm-assert/resources/images/camunda.png)&nbsp;camunda-bpm-assert User Guide for BPMN
+# Camunda BPM Assert User Guide for BPMN
```

Comment:
Are you trying to get a t-shirt? :smile:

Reply:
Damn it, you got me there!